front/app/components/VoteControl/index.tsx

```diff
@@ -233,22 +218,12 @@ const Vote = styled.button<IVoteComponent>`
     flex-direction: row-reverse;
   `}
 
-  &:not(.compact).voteClick ${VoteIconContainer} {
+  &.voteClick ${VoteIconContainer} {
```

Comment:
Removed 'not compact' because there's no compact anymore, so everything will be 'not compact' by default.